src/ng-select/lib/ng-dropdown-panel.component.ts

```diff
@@ -23,424 +24,425 @@ import { animationFrameScheduler, asapScheduler, fromEvent, merge, Subject } fro
 import { auditTime, takeUntil } from 'rxjs/operators';
 import { NgDropdownPanelService, PanelDimensions } from './ng-dropdown-panel.service';
 
-import { DropdownPosition } from './ng-select.types';
-import { NgOption } from './ng-select.types';
+import { DropdownPosition, NgOption } from './ng-select.types';
 import { isDefined } from './value-utils';
 
 const CSS_POSITIONS: Readonly<string[]> = ['top', 'right', 'bottom', 'left'];
 const SCROLL_SCHEDULER = typeof requestAnimationFrame !== 'undefined' ? animationFrameScheduler : asapScheduler;
 
 @Component({
-    changeDetection: ChangeDetectionStrategy.OnPush,
-    encapsulation: ViewEncapsulation.None,
-    selector: 'ng-dropdown-panel',
-    template: `
-        <div *ngIf="headerTemplate" class="ng-dropdown-header">
-            <ng-container [ngTemplateOutlet]="headerTemplate" [ngTemplateOutletContext]="{ searchTerm: filterValue }"></ng-container>
-        </div>
-        <div #scroll class="ng-dropdown-panel-items scroll-host">
-            <div #padding [class.total-padding]="virtualScroll"></div>
-            <div #content [class.scrollable-content]="virtualScroll && items.length">
-                <ng-content></ng-content>
-            </div>
-        </div>
-        <div *ngIf="footerTemplate" class="ng-dropdown-footer">
-            <ng-container [ngTemplateOutlet]="footerTemplate" [ngTemplateOutletContext]="{ searchTerm: filterValue }"></ng-container>
-        </div>
-    `
-    })
+	changeDetection: ChangeDetectionStrategy.OnPush,
+	encapsulation: ViewEncapsulation.None,
+	selector: 'ng-dropdown-panel',
+	standalone: true,
+	template: `
+		@if (headerTemplate) {
+			<div class="ng-dropdown-header">
+				<ng-container [ngTemplateOutlet]="headerTemplate" [ngTemplateOutletContext]="{ searchTerm: filterValue }" />
+			</div>
+		}
+		<div #scroll role="listbox" class="ng-dropdown-panel-items scroll-host">

```

Comment:
Does the addition of `role` create any new issues? It's technically in the right place now but I know we've made other holistic changes to the accessibility here.

Reply:
@geebru I noticed we weren't using it in the past but I haven't noticed any difference once this was added